src/blog/2023/07/how-to-deploy-a-basic-opc-ua-server-in-node-red.md

```diff
@@ -0,0 +1,686 @@
+---
+title: How to Deploy a Basic OPC-UA Server in Node-RED
+subtitle: OPC-UA Server Information Modeling in Node-RED
+description: Basic Information on OPC-UA and How to Deploy an Example OPC-UA Server Flow.
+date: 2023-07-12 
+authors: ["richard-meyer"]
+image: blog/2023/07/images/opc-ua-1/opc-ua-1-title-image.png
+tags:
+    - posts
+    - flowforge
+    - releases
+---
+
+This article is the first part of a series of OPC-UA content.  Here, we will explain some basic concepts of OPC-UA as they apply to building a server in Node-RED, then walk through and deploy an example OPC-UA Server.  
+
+<!--more-->
+
+## What is OPC-UA?
+
+OPC **U**nified **A**rchitecture (OPC UA) is an open, platform independent communication framework frequently utilized in industrial automation, and is considered one of the key protocol standards for Industry 4.0 and Industrial IoT (IIoT).  The standard is developed and maintained by a consortium called the OPC Foundation, with recognizable industry names such as Siemens, Honeywell, Microsoft, Beckhoff, SAP, Yokogawa, ABB, Rockwell, and Schneider Electric.
```

Comment:
Why are the U and A bolded?

Reply:
it's to emphasize what the UA comes from.  But not necessary.  I'll remove it.